source/develop/release-management/features/network/networking-api-security-groups.html.md

```diff
@@ -0,0 +1,352 @@
+---
+title: Networking API security group support
+category: feature
+authors: mdbarroso
+feature_name: security-group-support
+feature_modules: engine,ovirt-provider-ovn
+feature_status: In Development
+---
+
+
+# Networking API security group support
+
+
+## Summary
+
+This feature aims to implement networking API v2 security group support
+to external networks.
+
+By doing so, we allow fine-grained access control to - and from - the oVirt
+VMs attached to those external networks.
+
+The networking API v2 defines security groups as a white list of rules.
+That means, that by default, neither incoming nor outgoing traffic is
+allowed (from the VMs perspective).
+
+A security group is applied to a logical port - a *logical switch port*
+in the OVN model.
+
+### Owner
+
+* Feature Owner: [Miguel Duarte Barroso](https://github.com/maiqueb) (mbarroso on IRC)
+* Email: <mdbarroso@redhat.com>
+
+### Terminology
+
+* Networking API: openstack RESTful API
+* OVN: Open Virtual Network. OVN is an OVS (Open vSwitch) extension that
+brings Software Defined Networking to OVS.
+* oVirt OVN provider: a proxy that implements a subset of the
+[networking API v2](https://developer.openstack.org/api-ref/network/v2/)
+and interacts with OVN.
+* Security group: container for security group **rules**; networking API
+entity
+* Security group rules: specify the access to a network resource;
+networking API entity
+
+### Benefit to oVirt
+
+Currently, all incoming or outgoing traffic to oVirt VMs is allowed,
+which - at best - is a security risk. This lax practice could lead to
+security breaches and information theft in the oVirt clusters.
+
+Security groups, which are a traffic white list, help protect against
+these issues by specifying which traffic is allowed to *and* from the
+resources.
+
+## Objectives
+
+* Provide a backwards compatible security group implementation, using OVN
+as the backend. If possible, no additional backend / database should be
+used.
+
+* When possible, Openstack neutron's behavior should be mimicked: by
+default, all VM incoming IP traffic is dropped, and all outgoing IP
+traffic is allowed.
+
+* The security group feature should be configurable, and activated by
+default - this means an option for using oVirt's external networks
+without security group support should be possible.
+
+* If the external network has a subnet defined, DHCP traffic is allowed to
+/ from the VM.
+
+### Requirements
+
```

Comment:
The following requests will be implemented:

Reply:
I removed the list of requests, since, all in all, having it was just dumb; those were just the methods allowed in the networking api specification.